tests/test_query_tellor_rng.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from dataclasses import dataclass
+
+from telliot_core.dtypes.bytes32_type import Bytes32Type
+from telliot_core.dtypes.value_type import ValueType
+from telliot_core.queries.abi_query import AbiQuery
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
For the test, use one of the other query type tests as an example (e.g. `test_query_morphware.py`). This test file has the same content as `tellor_rng.py`